composer.json

```diff
@@ -1,10 +1,8 @@
 {
-    "name":     "jsvrcek/ics",
-    "type":     "library",
+    "name": "jsvrcek/ics",
     "description": "abstraction layer for creating multi-byte safe RFC 5545 compliant .ics files",
     "keywords": ["ical", "calendar", "export", ".ics", "RFC 5545", "multi-byte safe"],
     "license": "MIT",
-    "homepage": "https://github.com/jasvrcek/ICS",
```

Comment:
@vinkla thanks for working on this. Before I merge the pull request, is there a reason to remove the link to the github repo?